docs/how-to.md

```diff
@@ -31,6 +31,18 @@ comprehensive) overview of all that is possible with the package.
 associated with `kubernetes-el`, as well as terminate all associated
 background processes, e.g. those for querying resources.
 
+## Interface Customization
+
+### Selecting views
+| Keybinding | Interactive function                   |
+|:-----------|:---------------------------------------|
+| `v`        | `M-x kubernetes-overview-select-views` |
+
+Through `kubernetes-overview-select-views` (`v`) a list of views can be selected. The selected views will be rendered above one another in the overview buffer. Additional (custom) views can be defined through the customizable `kubernetes-overview-custom-views-alist` variable.
+
+![Selecting views](./img/views-select.gif)
```

Comment:
I think you missed to add this file?

Reply:
Ah yes, you're right. Sorry for that. I've added it now.